vcbuild.bat

```diff
@@ -49,7 +48,6 @@ if /i "%1"=="clean"         set target=Clean&goto arg-ok
 if /i "%1"=="ia32"          set target_arch=x86&goto arg-ok
 if /i "%1"=="x86"           set target_arch=x86&goto arg-ok
 if /i "%1"=="x64"           set target_arch=x64&goto arg-ok
-if /i "%1"=="vc2015"        set target_env=vc2015&goto arg-ok
```

Comment:
This should be kept. `vcbuild.bat` should search for VS2017 and, only if that is not found, VS2015. But a user with both installed should be able to use `vc2015/2017` to specifically select the version to use or fail if that version is not found. This mechanism was introduced in https://github.com/nodejs/node/pull/4645, I don't think it should be removed.